src/searches/issueAttachments.ts

```diff
@@ -28,72 +27,73 @@ interface issueAttachmentsRequestResponse {
           }
           attachments {
             nodes {
+              id,
```

Comment:
the URL is missing is it expected ?

Reply:
Definitely not, went too fast 🤦 thanks for the catch